scripts/zkai_d128_full_block_accumulator_backend_gate.py

```
@@ -0,0 +1,954 @@
+import csv
+import hashlib
+import importlib.util
+import io
+import json
+import os
+import pathlib
+import stat as stat_module
+import sys
+import tempfile
+from functools import lru_cache
+from typing import Any, Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)